src/shared/services/WalletService/types.ts

```diff
@@ -1,13 +1,12 @@
-import { HashConnectConnectionState } from "hashconnect/dist/esm/types";
-import { HashConnectTypes, MessageTypes } from "hashconnect";
+import { HashConnectConnectionState } from "hashconnect/dist/types";
 
 interface HashConnectEventHandlers {
-  handleFoundExtensionEvent: (walletMetadata: HashConnectTypes.WalletMetadata) => void;
-  handlePairingEvent: (approvePairing: MessageTypes.ApprovePairing) => void;
-  handleAcknowledgeMessageEvent: (acknowledgeData: MessageTypes.Acknowledge) => void;
+  handleFoundExtensionEvent: (walletMetadata: any) => void;
+  handlePairingEvent: (approvePairing: any) => void;
```

Comment:
> check the types the lib now uses for those events, maybe we can use them instead of `any`
> 
> ```
> hashconnect.pairingEvent.on(handlePairingEvent);
> 
>  HashConnect
> pairingEvent: Event<SessionData>
> ```

Well, i see, for me it's just makes sense to remove those handlers completely as they not exposed by `hashconnect` anymore.